ci/azure/windows.yml

```diff
@@ -19,6 +19,8 @@ jobs:
   steps:
     - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
       displayName: Add conda to PATH
+    - script: conda update -n base conda
```

Comment:
nitpick: it's nice to have `-q` here (and in the `conda env create` below) so that we don't have long logs just showing the download progress.